app/app.go

```diff
@@ -43,7 +43,7 @@ type Interface interface {
 	Features() *Features // Get this application's enterprise features
 }
 
-var pkgLogger logger.LoggerI
+var pkgLogger logger.Logger
 
 func Init() {
 	pkgLogger = logger.NewLogger().Child("app")
```

Comment:
I believe we could do the following:

```go
	var pkgLogger = logger.NewLogger().Child("app")
```

And avoid having an `Init()` method here; it doesn't provide any more benefit as there is always a default logger initialised.